uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1025.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1025 {
+
+  /**
+   * Deletes some demographics fields where they are not in use
+   *
+   * @return bool
+   */
+  public function upgrade_1025() {
+
+    $this->up1025_deleteExtendDemographicFields([
+      'Ethnicity',
+      'Religion',
+      'Sexual_Orientation',
+    ]);
+
+    return TRUE;
+  }
+
+  /**
+   * Deletes Custom Demographic Fields only if they are not used
+   *
+   * @param array $fieldsToDelete
+   */
+  private function up1025_deleteExtendDemographicFields($fieldsToDelete) {
+    $customGroup = civicrm_api3('CustomGroup', 'get', [
+      'name' =>'Extended_Demographics',
```

Comment:
Tiny, tiny thing, but you're missing a space after `=>`

Reply:
Corrected before merge